.github/workflows/mac_release.yml

```diff
@@ -8,22 +8,30 @@ on:
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   BUILD_TYPE: Release
-  FC: /usr/local/bin/gfortran-9
+  FC: gfortran-11
```

Comment:
macos-12 only has gfortran-11. The two others have it too, so use that.

https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#language-and-runtime

Reply:
This gfortran goes up to 11.